config/initializers/breakers.rb

```diff
@@ -11,62 +12,59 @@
 require 'evss/documents_service'
 require 'evss/letters/service'
 require 'evss/gi_bill_status/service'
+require 'evss/letters/service'
 require 'facilities/bulk_configuration'
 require 'gi/configuration'
 require 'hca/configuration'
 require 'mhv_ac/configuration'
 require 'mvi/configuration'
+require 'okta/configuration'
 require 'preneeds/configuration'
 require 'rx/configuration'
 require 'sm/configuration'
 require 'search/configuration'
-require 'okta/configuration'
-
-require 'evss/claims_service'
-require 'evss/common_service'
-require 'evss/documents_service'
-require 'evss/letters/service'
 
 # Read the redis config, create a connection and a namespace for breakers
 redis_config = Rails.application.config_for(:redis).freeze
 redis = Redis.new(redis_config['redis'])
 redis_namespace = Redis::Namespace.new('breakers', redis: redis)
 
+
 services = [
   Appeals::Configuration.instance.breakers_service,
-  Rx::Configuration.instance.breakers_service,
```

Comment:
This one got removed but not added -- on purpose?

Reply:
It's there -> just hard to see because a-z PRs are always messy. Will separate.